utils/utils.ts

```diff
@@ -97,3 +105,18 @@ export const parseUserPick = (userId: IUser['id'], teamId: string): IUserPick =>
 
   return parsedData;
 };
+
+/**
+ * Get the list of leagues the user is a part of
+ *
+ * @return {ILeague | Error}
+ */
+export const getUserLeagues = async (
+  leagues: IUser['league'],
+): Promise<ILeague[]> => {
+  const userLeagues = leagues.map((league) => {
+    return getCurrentLeague(league);
+  });
+
+  return Promise.all(userLeagues);
```

Comment:
Does this actually work? `userLeagues` is an array of leagues. Each array value is fetched via `getCurrentLeague`. It returns to `userLeagues`, but does it append leagues or overrides the variable?

Reply:
Yes, this works.  The user has an array of league Id's that they are a part of.  You then have to go and fetch each league information and return it as an array so that we can populate the leagues page:

![CleanShot 2024-06-18 at 00 18 54](https://github.com/LetsGetTechnical/gridiron-survivor/assets/102450568/7b02608b-0896-4871-b256-90008f7270ca)

```
[
  {
    leagueId: '66311a210039f0532044',
    leagueName: 'The GIS OG Crew',
    logo: null,
    participants: [
      '66281d5ec5614f76bc91',
      '6632eaa0a65cd59045ed',
      '66269d85413eff63c89c',
      '664e6d5e002fb73808c0',
      '66634059003922d59bcb'
    ],
    survivors: [
      '66281d5ec5614f76bc91',
      '6632eaa0a65cd59045ed',
      '66269d85413eff63c89c',
      '664e6d5e002fb73808c0'
    ]
  }
]
```

Reply:
@shashilo also note that for server components we need console.log to see the data as it won't show up in the network tab of the browser.